discovery-provider/src/app.py

```diff
@@ -300,6 +301,9 @@ def default(self, o):
     app.register_blueprint(api_v1.bp)
     app.register_blueprint(api_v1.bp_full)
 
+    # Create challenges
+    create_challenges(app.db_session_manager)
```

Comment:
could we call this something like
create_new_challenges or create_non_existent_challenges
or something like that?